nio/client/base_client.py

```diff
@@ -899,6 +911,8 @@ def receive_response(self, response):
             self._handle_login(response)
         elif isinstance(response, LogoutResponse):
             self._handle_logout(response)
+        elif isinstance(response, RegisterResponse):
```

Comment:
You forgot to import `RegisterResponse` so almost all the tests are failing.

Reply:
fixed in next commit.